mailpoet/views/homepage.html

```diff
@@ -0,0 +1,5 @@
+<% extends 'layout.html' %>
+
+<% block content %>
+<h1>MailPoet Homepage</h1>
```

Comment:
Not sure if we leave this untranslated at this stage!

Reply:
This is just placeholder content. The whole homepage is hidden behind the feature flag. It is to be removed once we start adding real content. I decided to keep it there so that it is easier to verify that the homepage template loads.
I can remove it, but I wouldn't make it translatable since it will be removed soon. Do you think I should rather remove it?

Reply:
No need to change it if it's only a placeholder.